src/views/Home.vue

```diff
@@ -563,7 +563,8 @@
           <h3 class="title is-2 is-spaced">{{ $t("team.title") }}</h3>
           <p>{{ $t("team.body") }}</p>
           <div class="content team-content">
-            <div class="tile is-ancestor">
+            <!-- Flex wrap is manually required since tiles do not support flex otherwise -->
```

Comment:
What's the advantage of enforcing flex here? I think it's quite helpful to just let the CSS library deal with the layout. I'm a bit hesitant about interfering with it.